src/components/carousel/carousel.js

```diff
@@ -413,11 +433,11 @@ export default {
         setAttr(nextSlide, 'aria-hidden', 'false')
         currentSlide.tabIndex = -1
         nextSlide.tabIndex = -1
-        if (!this.isCycling) {
+        if (!this.isCycling && this.focus) {
           // Focus the next slide for screen readers if not in play mode
           nextSlide.tabIndex = 0
           this.$nextTick(() => {
-            nextSlide.focus()
```

Comment:
normally the focusing of the slide only happens when the slide is paused due to a mouseover.